tests/instruments/teledyne/test_teledyneMAUI_with_device.py

```diff
@@ -26,17 +26,15 @@
 
 from time import sleep
 
-from pymeasure.instruments.teledyne import TeledyneHDO6xxx
+from pymeasure.instruments.teledyne import TeledyneMAUI
 
 
-class TestTeledyneHDO6xxx:
+class TestTeledyneMAUI:
     """
-    Unit tests for TeledyneHDO6xxx class.
+    Unit tests for TeledyneMAUI class.
 
-    This test suite, needs the following setup to work properly:
-        - A TeledyneHDO6xxx device should be connected to the computer;
-        - The device's address must be set in the RESOURCE constant;
-        - A probe on Channel 1 must be connected to the Demo output of the oscilloscope.
```

Comment:
Isn't that line (a probe connected) still necessary or do you not use that demo output?

Reply:
No, it seems not necessary as I never connected that probe and all tests succeeded.

Reply:
Ok